crates/hyperswitch_interfaces/src/configs.rs

```diff
@@ -11,7 +11,7 @@ use serde::Deserialize;
 pub struct Connectors {
     pub aci: ConnectorParams,
     #[cfg(feature = "payouts")]
-    pub adyen: ConnectorParamsWithSecondaryBaseUrl,
+    pub adyen: ConnectorParamsWithThreeBaseUrls,
     pub adyenplatform: ConnectorParams,
     #[cfg(not(feature = "payouts"))]
     pub adyen: ConnectorParams,
```

Comment:
Irrespective of payout feature being enabled, we would need this url for disputes right?